extensions/elation/types/generic.ts

```diff
@@ -4,3 +4,13 @@ export interface Find<T> {
   previous: null | string
   results: T
 }
+
+/**
+ * Elation returns up to 100 results in a single query
+ */
+export interface ElationCollection<T> {
```

Comment:
ditto my comment above. `ElationCollection` and `Find` accomplish the same thing. Let's use just one. I like the way you defined it.